controller/WFCORE-4868_configuration-files-aliases.adoc

```diff
@@ -0,0 +1,100 @@
+---
+categories:
+ - controller
+---
+= [Community] Aliases for standard configuration files
+:author:            Michal Petrov
+:email:             mpetrov@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+Introduce aliases for standard configuration files
+
+This will allow users to reference the configuration by the suffix instead of the whole filename, e.g. `standalone.sh -c full` instead of `standalone.sh -c standalone-full.xml`
+The alias "abc" is shorthand for <baseName>-abc.xml where "baseName" is the basic name of a configuration file, i.e. "standalone" by default.
+
+In addition some of the basic configuration files could be referenced by an alias.
+
+.Full list of aliases:
+|===
+|  Configuration file | Alias
+
+| *-full-ha.xml          | fha
+
+| *-load-balancer.xml    | lb
+
+| *-microprofile.xml     | mp
+
+| *-microprofile-ha.xml  | mpha
+|===
+
+Special values for `-c` remain in effect, e.g. `-c boot` references the standalone.boot.xml file (if present) and it is not possible to reference a custom file named `standalone-boot.xml` this way.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFCORE-4868
+
+=== Stability Level
+* [ ] Experimental
+
+* [ ] Preview
+
+* [x] Community
+
+* [ ] default
+
+=== Dev Contacts
+
+* mailto:{email}[{author}]
+
```

Comment:
@michpetrov This section describes the feature team. 

Please add you as a developer, me as SME, and Emmanuel Hugonnet as an outside perspective person.